src/System.Security.Cryptography.Cng/pkg/System.Security.Cryptography.Cng.pkgproj

```diff
@@ -17,6 +17,7 @@
     <HarvestIncludePaths Include="ref/netstandard1.4;runtimes/win/lib/netstandard1.4" />
     <HarvestIncludePaths Include="ref/netstandard1.6;runtimes/win/lib/netstandard1.6;lib/netstandard1.6" />
     <HarvestIncludePaths Include="ref/netcoreapp2.0;runtimes/win/lib/netcoreapp2.0" />
+    <HarvestIncludePaths Include="ref/netcoreapp2.1;lib/netcoreapp2.1;runtimes/win/lib/netcoreapp2.1" />
```

Comment:
That sounds sensible.  I don't really know of a reason why I'd want to keep live-building 2.1.

Reply:
The only reason you might consider doing it is to avoid having the netcoreapp2.1 and netcoreapp3.0 assets in your package. We are essentially adding 2 netcoreapp targets in your package now.